index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <!-- navbar section -->
+        <section>
+            <header class="navbar">
+                <h4>Rabaya Rabu</h4>
```

Comment:
- [ ] The logo is usually an `a` tag which redirects the user to the home page. Kindly change the tag around your name to an `a` tag for a better user experience.
 - [ ] The logo is not perfectly aligned with the hamburger menu button on the right. Kindly fix the styling to match the Figma template.
 - Your design:
 - 
<img width="490" alt="image" src="https://user-images.githubusercontent.com/61919617/221186489-f5219e71-2b35-4ff3-942f-fcc07907a341.png">

Figma template:
<img width="397" alt="image" src="https://user-images.githubusercontent.com/61919617/221186541-65a0d97a-faa4-47bf-9d7d-64680b2a3efd.png">